.github/ISSUE_TEMPLATE/implement-new-feature.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Implement new feature
+about: Create a new featrue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Feature you want to implement
+What is a feature you want to implement?
```

Comment:
Maybe "_the_ feature" would be better, since you are talking about _the_ specific feature you want to implement _here_.